### PR TITLE
Misc. (mostly) early-game item sprites for Neodays

### PR DESCRIPTION
**Pull request by Golfavel**
#### Summary
NeoDays "Misc. early-game item sprites"

#### Content of the change

Adds various different (mostly) early-game item sprites for the "Neodays" tileset, including the large and flaking rocks, adhesive bandages and alcohol wipes, and even toothbrushes; list below
```
butter, plastic straw, bottle opener, chopsticks, dish towel, scrub brush, toilet brush, hair brush, dental floss, toilet plunger, lug wrench, penny, nickel, dime, half-dollar coin, dollar coin, toothbrush, electric toothbrush, dirty toothbrush, toilet paper, flaking rock, large rock, flint, alcohol wipes, adhesive bandages, comb, lice comb, shaving razor, dentures, curling iron, matchbox
reuses similar sprites for use on: dish soap, bow saw, polycarbonate sheet
```

#### Testing
Before:
miscsprites2
After:
miscsprites1

#### Additional information
I'm not completely satisfied with the brushes, I may try again on them later. Apologies for the messy commit history.

### CODE REVIEW

File: gfx/NeoDays/pngs_tiles_10x10/items/toilet_paper_0.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"id": ["toilet_paper"], "fg": ["toilet_paper"], "rotates": false, "bg": []}
```
Comment (Fris0uman) [inline gfx/NeoDays/pngs_tiles_10x10/items/toilet_paper_0.json:1]: All the jsons that associated one sprite with one id are unecessary. Here for exemple just name the sprite toilet_paper and remove the json. Same for the other json with only one id to one sprite

Comment (Golfavel) [reply gfx/NeoDays/pngs_tiles_10x10/items/toilet_paper_0.json]: Done; is there some new version of ``compose.py`` or argument that handles it like that? The version I've got seems to require them by default.

Comment (Fris0uman) [reply gfx/NeoDays/pngs_tiles_10x10/items/toilet_paper_0.json]: yes use the --use-all argument

**Comment by Fris0uman** [comment]
Looks good to me now. If you could also submit those sprite with black [0,0,0] instead of transparency to retrodays that would be nice

**Pull request merged**